mlir/include/mlir/Dialect/Mesh/IR/MeshOps.td

```diff
@@ -341,6 +342,68 @@ def Mesh_AllReduceOp : Mesh_CollectiveCommunicationOpBase<"all_reduce", [
   let hasCanonicalizer = 1;
 }
 
+def Mesh_AllScatterOp : Mesh_CollectiveCommunicationOpBase<"all_scatter", [
```

Comment:
I'm wondering if it's appropriate to include scatter in the operation's name, as scatter typically implies that the operation involves communication.

Reply:
I rename it to `all_slice`.